.env-simnet-local-sample

```diff
@@ -0,0 +1,60 @@
+###############################################################
+# Local SimNet Sparkswap Environment File
+#
+# This file contains settings that should be reviewed (and changed) when
+# running your broker daemon in a particular environment.
+#
+# These settings control the underlying functionality of sparkswapd and the
+# associated lightning implementations (engines).
+#
+###############################################################
+
+# Set the network that sparkswap and engines will work on.
+# Valid options are simnet, testnet, and mainnet.
```

Comment:
Maybe currently supported options: simnet, testnet and not currently supported: mainnet, Or just remove mainnet completely for now